src/index.html

```diff
@@ -1,14 +1,20 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Stop watch</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.scss" />
```

Comment:
```suggestion
    <link rel="stylesheet" href="./styles/main.scss">
```
fix everywhere in the head:
![image](https://github.com/mate-academy/layout_stop-watch/assets/86923089/dd1b3d2d-e33c-49d5-b61a-2b9a0edae0b6)

Reply:
I have set up Linter and Prettier this way purposely, that's why I can't fix this mistake. 
I have done my previous tasks with this problem..